src/main/java/at/ac/tuwien/kr/alpha/AnswerSetToXlsxWriter.java

```diff
@@ -1,5 +1,14 @@
 package at.ac.tuwien.kr.alpha;
 
+import at.ac.tuwien.kr.alpha.api.mapper.AnswerSetToObjectMapper;
```

Comment:
For the future, I suggest not to commit files in which only import statements have changed (to keep the pull request focused).

Reply:
I thought this was one of the `*`-import changes, but apparently it really is just a reordering of imports.